mkdocs.yml

```diff
@@ -71,8 +71,10 @@ markdown_extensions:
           format: !!python/name:pymdownx.superfences.fence_code_format
   - toc:
       permalink: "🔗"
-  - pymdownx.highlight
-  - pymdownx.superfences
```

Comment:
note: We already included `pymdownx.superfences` in the lines above (that are unchanged, and not displayed by default).